halo2_proofs/Cargo.toml

```
@@ -68,7 +68,8 @@ rand_core = { version = "0.6", default-features = false, features = ["getrandom"
 getrandom = { version = "0.2", features = ["js"] }
 
 [features]
-default = ["batch"]
+default = ["batch","multicore"]
+multicore = ["rayon"]
```

```suggestion
multicore = ["rayon/threads"]
```
(or `maybe-rayon/threads`, IDK if namespaced feature flags are resolved relative to the package name or the dependency name.)